proto/profile/network.proto

```diff
@@ -66,53 +92,15 @@ message NetworkConfigChange {
   // token provisioned via the controller.
   string server_token = 1;
 
-  // New or updated network configuration for one or more network ports.
-  // Ports not listed here will keep their current configuration unchanged.
-  NetworkConfig config_changes = 2;
-}
-
-// ConfigSource specifies where the network configuration originated.
-enum ConfigSource {
-  // The origin of the configuration is unknown.
-  CONFIG_SOURCE_UNSPECIFIED = 0;
-  // Received device configuration directly from the controller over HTTPS.
-  CONFIG_SOURCE_CONTROLLER = 1;
-  // Exported device config from the controller with the controller's signature,
-  // and installed via the Single-Use EVE installer as the device's initial configuration.
-  CONFIG_SOURCE_BOOTSTRAP = 2;
-  // Manually provided network-only config in JSON format and embedded into the EVE installer.
-  CONFIG_SOURCE_OVERRIDE = 3;
-  // Exported device config from the controller with the controller's signature,
-  // imported into the Local Operator Console (LOC), and fetched by the device over HTTPS.
-  CONFIG_SOURCE_LOC = 4;
-  // Created using a Local Profile Server (LPS) application and fetched
-  // by the device using the LPS API (currently HTTP-only).
-  CONFIG_SOURCE_LPS = 5;
-}
+  // Network configuration changes to be applied to one or more ports.
```

Comment:
"changes" and "NetworkConfigChange" sounds imperative and not declarative. Change since when?
See below for an alternative but if we go with a declarative approach then we shouldn't use "change" anywhere but instead use "local" and/or "override".

Reply:
OK, done. And yes, there is a per-port `error_message` defined inside `NetworkInfo`, which EVE publishes to LPS. This error message will be used to report permission, validation, etc. errors.